--- sample-scripts/openvpn.init.orig	2003-06-11 17:43:02.000000000 +0200
+++ sample-scripts/openvpn.init	2003-06-11 17:44:57.000000000 +0200
@@ -116,7 +116,7 @@ case "$1" in
 		. $bn.sh
 	    fi
 	    rm -f $piddir/$bn.pid
-	    $openvpn --daemon --writepid $piddir/$bn.pid --config $c --cd $work
+	    $openvpn --user openvpn --group openvpn --daemon --writepid $piddir/$bn.pid --config $c --cd $work
 	    if [ $? = 0 ]; then
 		successes=1
 	    else
