--- sample-scripts/openvpn.init	2011-05-21 11:04:02.020210798 +0200
+++ sample-scripts/openvpn.init.oden	2011-05-21 11:05:24.041210714 +0200
@@ -10,6 +10,17 @@
 #              certification features of the OpenSSL library to securely \
 #              tunnel IP networks over a single UDP port.
 #
+### BEGIN INIT INFO
+# Provides: openvpn
+# Required-Start: $network
+# Required-Stop: $network
+# Default-Start: 3 4 5
+# Short-Description: OpenVPN tunneling application
+# Description: OpenVPN is a robust and highly flexible tunneling application that
+#              uses all of the encryption, authentication, and certification features
+#              of the OpenSSL library to securely tunnel IP networks over a single
+#              UDP port.
+### END INIT INFO
 
 # Contributed to the OpenVPN project by
 # Douglas Keller <doug@voidstar.dyndns.org>
