diff -Naurp openvpn-2.3.0/sample/sample-config-files/client.conf openvpn-2.3.0.user.nobody/sample/sample-config-files/client.conf
--- openvpn-2.3.0/sample/sample-config-files/client.conf	2012-09-10 11:01:08.000000000 -0400
+++ openvpn-2.3.0.user.nobody/sample/sample-config-files/client.conf	2013-03-20 12:04:40.844586861 -0400
@@ -58,8 +58,8 @@ resolv-retry infinite
 nobind
 
 # Downgrade privileges after initialization (non-Windows only)
-;user nobody
-;group nobody
+;user openvpn
+;group openvpn
 
 # Try to preserve some state across restarts.
 persist-key
diff -Naurp openvpn-2.3.0/sample/sample-config-files/server.conf openvpn-2.3.0.user.nobody/sample/sample-config-files/server.conf
--- openvpn-2.3.0/sample/sample-config-files/server.conf	2012-09-10 11:01:08.000000000 -0400
+++ openvpn-2.3.0.user.nobody/sample/sample-config-files/server.conf	2013-03-20 12:03:44.992285111 -0400
@@ -259,8 +259,8 @@ comp-lzo
 #
 # You can uncomment this out on
 # non-Windows systems.
-;user nobody
-;group nobody
+;user openvpn
+;group openvpn
 
 # The persist options will try to avoid
 # accessing certain resources on restart
diff -Naurp openvpn-2.3.0/sample/sample-config-files/static-home.conf openvpn-2.3.0.user.nobody/sample/sample-config-files/static-home.conf
--- openvpn-2.3.0/sample/sample-config-files/static-home.conf	2012-09-10 11:01:08.000000000 -0400
+++ openvpn-2.3.0.user.nobody/sample/sample-config-files/static-home.conf	2013-03-20 12:05:28.793987411 -0400
@@ -37,10 +37,10 @@ secret static.key
 ; port 1194
 
 # Downgrade UID and GID to
-# "nobody" after initialization
+# "openvpn" after initialization
 # for extra security.
-; user nobody
-; group nobody
+; user openvpn
+; group openvpn
 
 # If you built OpenVPN with
 # LZO compression, uncomment
diff -Naurp openvpn-2.3.0/sample/sample-config-files/static-office.conf openvpn-2.3.0.user.nobody/sample/sample-config-files/static-office.conf
--- openvpn-2.3.0/sample/sample-config-files/static-office.conf	2012-09-10 11:01:08.000000000 -0400
+++ openvpn-2.3.0.user.nobody/sample/sample-config-files/static-office.conf	2013-03-20 12:05:45.125783236 -0400
@@ -34,10 +34,10 @@ secret static.key
 ; port 1194
 
 # Downgrade UID and GID to
-# "nobody" after initialization
+# "openvpn" after initialization
 # for extra security.
-; user nobody
-; group nobody
+; user openvpn
+; group openvpn
 
 # If you built OpenVPN with
 # LZO compression, uncomment
diff -Naurp openvpn-2.3.0/sample/sample-config-files/tls-home.conf openvpn-2.3.0.user.nobody/sample/sample-config-files/tls-home.conf
--- openvpn-2.3.0/sample/sample-config-files/tls-home.conf	2012-09-10 11:01:08.000000000 -0400
+++ openvpn-2.3.0.user.nobody/sample/sample-config-files/tls-home.conf	2013-03-20 12:05:11.359205376 -0400
@@ -48,10 +48,10 @@ key home.key
 ; port 1194
 
 # Downgrade UID and GID to
-# "nobody" after initialization
+# "openvpn" after initialization
 # for extra security.
-; user nobody
-; group nobody
+; user openvpn
+; group openvpn
 
 # If you built OpenVPN with
 # LZO compression, uncomment
diff -Naurp openvpn-2.3.0/sample/sample-config-files/tls-office.conf openvpn-2.3.0.user.nobody/sample/sample-config-files/tls-office.conf
--- openvpn-2.3.0/sample/sample-config-files/tls-office.conf	2012-09-10 11:01:08.000000000 -0400
+++ openvpn-2.3.0.user.nobody/sample/sample-config-files/tls-office.conf	2013-03-20 12:06:14.761412740 -0400
@@ -48,10 +48,10 @@ key office.key
 ; port 1194
 
 # Downgrade UID and GID to
-# "nobody" after initialization
+# "openvpn" after initialization
 # for extra security.
-; user nobody
-; group nobody
+; user openvpn
+; group openvpn
 
 # If you built OpenVPN with
 # LZO compression, uncomment
diff -Naurp openvpn-2.3.0/sample/sample-config-files/xinetd-client-config openvpn-2.3.0.user.nobody/sample/sample-config-files/xinetd-client-config
--- openvpn-2.3.0/sample/sample-config-files/xinetd-client-config	2012-09-10 11:01:08.000000000 -0400
+++ openvpn-2.3.0.user.nobody/sample/sample-config-files/xinetd-client-config	2013-03-20 12:03:31.731450895 -0400
@@ -6,6 +6,6 @@ dev tun
 ifconfig 10.4.0.1 10.4.0.2
 remote my-server
 port 1194
-user nobody
+user openvpn
 secret /root/openvpn/key
 inactive 600
diff -Naurp openvpn-2.3.0/sample/sample-config-files/xinetd-server-config openvpn-2.3.0.user.nobody/sample/sample-config-files/xinetd-server-config
--- openvpn-2.3.0/sample/sample-config-files/xinetd-server-config	2012-09-10 11:01:08.000000000 -0400
+++ openvpn-2.3.0.user.nobody/sample/sample-config-files/xinetd-server-config	2013-03-20 12:04:51.713450982 -0400
@@ -21,5 +21,5 @@ service openvpn_1
         wait            = yes
         user            = root
         server          = /root/openvpn/openvpn
-        server_args     = --inetd --dev tun --ifconfig 10.4.0.2 10.4.0.1 --secret /root/openvpn/key --inactive 600 --user nobody
+        server_args     = --inetd --dev tun --ifconfig 10.4.0.2 10.4.0.1 --secret /root/openvpn/key --inactive 600 --user openvpn
 }
