diff --git a/src/openvpn/options.c b/src/openvpn/options.c
index 9fef394..06bba91 100644
--- a/src/openvpn/options.c
+++ b/src/openvpn/options.c
@@ -5596,7 +5596,7 @@ add_option(struct options *options,
         {
             int z;
             const char *name = NULL;
-            const char *opterr = "when --inetd is used with two parameters, one of them must be 'wait' or 'nowait' and the other must be a daemon name to use for system logging";
+            const char opterr[] = "when --inetd is used with two parameters, one of them must be 'wait' or 'nowait' and the other must be a daemon name to use for system logging";
 
             options->inetd = -1;
 
