--- openvpn-2.2.1/configure.ac.bero	2013-01-16 01:51:41.375958664 +0100
+++ openvpn-2.2.1/configure.ac	2013-01-16 01:51:57.533764770 +0100
@@ -27,12 +27,12 @@ AC_PREREQ(2.50)
 
 m4_include(version.m4)
 AC_INIT([OpenVPN], [PRODUCT_VERSION], [openvpn-users@lists.sourceforge.net], [openvpn])
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_SRCDIR(syshead.h)
 
 dnl Guess host type.
 AC_CANONICAL_HOST
-AM_INIT_AUTOMAKE(openvpn, [$PACKAGE_VERSION])
+AM_INIT_AUTOMAKE
 
 AC_ARG_WITH(cygwin-native,
 	[  --with-cygwin-native              Compile native win32],
